acceptance/new-user/join-button.test.js

```diff
@@ -0,0 +1,64 @@
+/** @jest-environment jsdom */
+const puppeteer = require("puppeteer");
+const URLS = require("../../constants/urls");
+const { JOIN } = URLS;
+const { delay } = require("../../constants/utils");
+
+let browser, page;
+
+async function confirmAlerts() {
+  await page.on("dialog", async (dialog) => {
+    await dialog.accept();
+
+    delay(2000);
+    await page.waitForNavigation();
+    expect(page.url()).toMatch(/login/);
+  });
+}
+
+/* check status*/
+async function checkResponse() {
```

Comment:
since this is called only once and we are not reusing it. I don't see any point in creating it as a separate function

Reply:
I can't change that, it is done like this for proper asynchronous behavior, otherwise, I have to change the whole test and it was also making the code messy.

Reply:
you can check the navigation, it is working properly in both headless true and false with all condition met

Reply:
The value of headless in Puppeteer tests determines whether the browser runs in a headless mode or not. When set to `true`, the browser operates without a visible UI, while setting it to `false` enables the browser to run with a graphical interface. However, regardless of the headless setting, the functionality and behavior of the navigation in Puppeteer tests remain unaffected.

Reply:
will this be also taken care in the new issue ticket you created ?

Reply:
No, I will add this @heyrandhir